cli/cli.go

```diff
@@ -101,7 +107,9 @@ func (cli *CLI) Run(ctx context.Context, cmdArgs ...string) error {
 		}
 		return cli.RunConvert(ctx, args)
 	case args.Bool("index"):
-		return cli.RunIndex(ctx, args)
+		hash, err := cli.RunIndex(ctx, args)
+		cli.stdout.Write([]byte(hash))
```

Comment:
The error should be checked before printing the hash.

Reply:
Won't it be an empty string, that is zero bytes, if the err is non-nil?